lerna.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.5.0-dev",
+  "version": "1.5.2",
```

Comment:
Is this supposed to be `1.6.0-dev` on develop branch?

Reply:
Yes it should be 1.6.0-dev.

Reply:
Yes, we should always bump the version to `-dev` on `develop`.